discoverer/README.md

```diff
@@ -0,0 +1,38 @@
+SeaClouds Discoverer
+====================
+
+The SeaClouds Discoverer module provides information about cloud offering used by SeaCloudsPlatform.
+
+# Installing and running SeaClouds Discoverer
+The easiest way to install and run this component is explained in SeaClouds [README.md](../README.md).
+
+## Requirements
+Running SeaClouds Discoverer only requires Java 7 (or greater) installed on the target machine. Among with the software requirements, it also needs:
+
+- A discovererconf.yml configuration file.
+- A running MongoDB service
```

Comment:
@PaoloCifariello is currently working on the documentation, and the new version will explain the usage of MongoDB.

Again, the DB was requested by @fradandria in Malaga. We added it to ensure persistency and consistency of offerings, especially in the case that SeaClouds is deployed on a cloud platform and multiple instances of the Discoverer have to access the DB. Furthermore, by making a snapshot of the DB we can also migrate it on a different platforms.

We decided to use MongoDB, since we were left free in choosing the technology (also this was discussed in the meeting in Malaga).